src/components/views/ViewCampaign.jsx

```diff
@@ -339,7 +339,20 @@ class ViewCampaign extends Component {
                       <div className="card content-card ">
                         <div className="card-body content">{this.renderDescription()}</div>
                       </div>
-
+                      <div className="text-center">
+                        <div style={{ transform: 'translateY(-50%)' }}>
+                          <DonateButton
```

Comment:
@mahdine Add the disable auto popup property to this DonateButton component, we need just one Donate Button with auto popup one page